x/eibc/client/cli/query_command_orders.go

```diff
@@ -68,11 +93,62 @@ func CmdListDemandOrdersByStatus() *cobra.Command {
 				return fmt.Errorf("failed to fetch demand orders: %w", err)
 			}
 
-			return clientCtx.PrintProto(res)
+			out, err := cmd.Flags().GetString("output")
+			if err == nil && out == "json" {
+				return clientCtx.PrintProto(res)
+			}
+
+			count := 0
+
+			for _, o := range res.DemandOrders {
+				count++
+
+				fmt.Printf(`
+-id:		%s
+  recipient:	%s
+  price:	%s
+  fee:		%s
+  rollapp_id:	%s
+  status:	%s
+  packet_key:	%s
+  fulfiller:	%s
+`,
+					o.Id, o.Recipient, parseAndFormat(o.Price.String()), parseAndFormat(o.Fee.String()), o.RollappId,
+					o.TrackingPacketStatus.String(), strings.TrimSpace(o.TrackingPacketKey), o.FulfillerAddress)
+			}
+
+			fmt.Printf("\ncount: %d; ts: %s\n", count, time.Now().String())
+
+			return nil
 		},
 	}
 
+	cmd.Flags().StringP("rollapp", "r", "", "Rollapp ID")
+	cmd.Flags().StringP("recipient", "c", "", "Recipient address")
+	cmd.Flags().StringP("type", "t", "", "Packet type")
+	cmd.Flags().StringP("denom", "d", "", "Denom")
+	cmd.Flags().StringP("fulfilled", "f", "", "Filter by fulfillment status")
+	cmd.Flags().StringP("fulfiller", "a", "", "Filter by fulfiller address")
+	cmd.Flags().Int32P("limit", "l", 0, "Limit orders to display")
 	flags.AddQueryFlagsToCmd(cmd)
 
 	return cmd
 }
+
+func parseAndFormat(amountStr string) string {
+	coin, err := sdk.ParseCoinNormalized(amountStr)
+	if err != nil {
+		panic(err)
+	}
+	if coin.Denom != "adym" {
+		return coin.String()
+	}
+	return formatAmount(coin.Amount.String()) + coin.Denom
+}
+
+func formatAmount(numStr string) string {
+	if len(numStr) <= 18 {
+		return "0," + strings.Repeat("0", 18-len(numStr)) + numStr
+	}
+	return numStr[:len(numStr)-18] + "," + numStr[len(numStr)-18:]
```

Comment:
what is this doing?

Reply:
the idea was to handle adym and separate the decimal part. This was originally for my own convenience, but I am not sure if everyone will want this

Reply:
ok but it should have a doc string so readers dont have to reverse engineer it

Reply:
anyways can you check if there is a simpler way or existing solution for this?
this kind of manual code looks sus
https://stackoverflow.com/a/51546906

Reply:
and it seems like a generally good fit for a 'denom utility package' in https://github.com/dymensionxyz/sdk-utils